packages/application/style/base.css

```diff
@@ -80,6 +80,14 @@ body[data-notebook='edit'] #spacer-widget-bottom {
   box-shadow: var(--jp-elevation-z4);
 }
 
+#main-panel .jp-TreePanel {
+  padding: 0rem 2rem;
```

Comment:
@jtpio I added paddings as the drop shadows on the left and right sides get hidden on full length.

Do you suggest to add responsive media queries for the same?

Reply:
Right we can probably keep the padding by default, but it's nice to use the full width on mobile devices.

For example the notebook view defines a media query here to save some vertical space: 

https://github.com/jupyter/notebook/blob/fd45f31f983097e1217d4a42488aebef9dfb3d24/packages/notebook-extension/style/base.css#L111-L123

Reply:
@jtpio Agreed! Something like the below should resolve this:

```
@media only screen and (max-width: 760px) {
  #main-panel .jp-TreePanel {
    padding: 0rem;
  }
}

#main-panel .jp-TreePanel {
  padding: 0rem 2rem;
}
```

Reply:
Hey @jtpio. It seems the width issue should be resolved. However the tests seem to give out another error. Could you guide me on that?

Reply:
Looks like the width is still smaller: 

![image](https://github.com/jupyter/notebook/assets/591645/565a37c1-9b98-4452-9ad6-3c1df1fa1276)


Could it be that the media query should be placed after the other rule so it can take effect?

Reply:
@jtpio I ran ui-tests locally. While most of them pass, the FileBrowser tests seem to fail:

![ui-tests](https://github.com/jupyter/notebook/assets/89381263/2cb3c416-3eb7-4cd4-9e66-cae7c4c8ae46)

Reply:
The UI tests might fail locally because the snapshots are slightly different than the ones on CI.

If you re-order the CSS and rebuild the application locally, what happens when resizing the window?

Reply:
@jtpio On window resize, only the vertical padding snaps at breakpoint, the horizontal ones always require a refresh.

Reply:
Ah ok then maybe we need to use the `body[data-format='mobile']` selector, like here:

https://github.com/jupyter/notebook/blob/fd45f31f983097e1217d4a42488aebef9dfb3d24/packages/notebook-extension/style/base.css#L134-L136

Reply:
@jtpio So I added those changes and in the local tests, the results look awesome:

![tests_passed](https://github.com/jupyter/notebook/assets/89381263/5a293c91-f8e9-482e-addf-ffcecd0b065a)

But the tests here are failing. The problem I had noticed was the responsiveness that does not kick in on window resize.

Reply:
> The problem I had noticed was the responsiveness that does not kick in on window resize, until page refresh.

Ah I wonder if this could then be the same issue as https://github.com/jupyter/notebook/issues/6553 and https://github.com/jupyter/notebook/pull/6887. 

Maybe we can try using the same trick as in this test:

https://github.com/jupyter/notebook/blob/fd45f31f983097e1217d4a42488aebef9dfb3d24/ui-tests/test/mobile.spec.ts#L46-L47

Above here:

https://github.com/jupyter/notebook/blob/fd45f31f983097e1217d4a42488aebef9dfb3d24/ui-tests/test/mobile.spec.ts#L15

If that helps make the UI tests pass on CI then we can investigate this separately in https://github.com/jupyter/notebook/pull/6887.

Reply:
Ah I think I was looking at the wrong version because the workaround is already there:

https://github.com/jupyter/notebook/blob/fd45f31f983097e1217d4a42488aebef9dfb3d24/ui-tests/test/mobile.spec.ts#L15-L28

Reply:
@jtpio I worked out a solution that passes through the mobile layout tests!😄

The changes allow switching between mobile and desktop views without refreshing. Now, the only tests failing seem to be the `Should be persisted after reloading the page`.

I guess the rebase should help resolve this if I'm correct.

Reply:
> Now, the only tests failing seem to be the `Should be persisted after reloading the page`.

Nice. yes that's because there are a few changes to the snapshots with the new box shadow. I'll trigger a new snapshot update by the bot.

Reply:
That makes sense. Glad to see them working now😄